From a8fe4ce53f23667684883000f9dbec1b4c149f76 Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Thu, 12 May 2016 22:04:24 +0200
Subject: systemd: syslog.target is obsolete

After=syslog.target is unnecessary by now because syslog is socket-activated and will therefore be started when needed.

Ref: https://lintian.debian.org/tags/systemd-service-file-refers-to-obsolete-target.html
---
 packaging/systemd/nmb.service     | 2 +-
 packaging/systemd/samba.service   | 2 +-
 packaging/systemd/smb.service     | 2 +-
 packaging/systemd/winbind.service | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/packaging/systemd/nmb.service b/packaging/systemd/nmb.service
index 3d71a7d..98d8080 100644
--- a/packaging/systemd/nmb.service
+++ b/packaging/systemd/nmb.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Samba NMB Daemon
-After=syslog.target network.target
+After=network.target
 
 [Service]
 Type=notify
diff --git a/packaging/systemd/samba.service b/packaging/systemd/samba.service
index 824f89c..83e0daf 100644
--- a/packaging/systemd/samba.service
+++ b/packaging/systemd/samba.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Samba AD Daemon
-After=syslog.target network.target
+After=network.target
 
 [Service]
 Type=notify
diff --git a/packaging/systemd/smb.service b/packaging/systemd/smb.service
index 9810891..949f798 100644
--- a/packaging/systemd/smb.service
+++ b/packaging/systemd/smb.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Samba SMB Daemon
-After=syslog.target network.target nmb.service winbind.service
+After=network.target nmb.service winbind.service
 
 [Service]
 Type=notify
diff --git a/packaging/systemd/winbind.service b/packaging/systemd/winbind.service
index f711a17..26c6752 100644
--- a/packaging/systemd/winbind.service
+++ b/packaging/systemd/winbind.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Samba Winbind Daemon
-After=syslog.target network.target nmb.service
+After=network.target nmb.service
 
 [Service]
 Type=notify
-- 
2.8.1

